src/service/main.ts

```diff
@@ -406,19 +409,18 @@ var runTradingSystem = (classes: SimulationClasses) : Q.Promise<boolean> => {
             }, moment.duration(1000));
             return completedSuccessfully.promise;
         };
-    
+
         // event looped blocked timer
         var start = process.hrtime();
         var interval = 100;
         setInterval(() => {
             var delta = process.hrtime(start);
             var ms = (delta[0] * 1e9 + delta[1]) / 1e6;
             var n = ms - interval;
-            if (n > 25)
-                mainLog.info("Event looped blocked for " + Utils.roundFloat(n) + "ms");
+            ////if (n > 25) mainLog.info("Event looped blocked for " + Utils.roundFloat(n) + "ms");
```

Comment:
Please do not remove this logging. It's quite useful to determine if there is stuff clogging up the event loop.